### ISSUE TITLE
Make user follow discussion by commenting

### ISSUE THREAD

**Issue by VytorCalixto** [PETComputacaoUFPR/core#1]
Right now, you only follow a discussion if you press a button (the follow button). I think the default behavior should be to follow a discussion that you:
1. Create
2. Comment

I think that would be nice to follow tags (but maybe that's another issue).

**Comment by dhpaulino** [comment]
I agree, but I think that when a discussion is created a comment is created too.

**Comment by dhpaulino** [comment]
Yeah, seems like it does at [here]https://github.com/PETComputacaoUFPR/core/blob/master/src/Core/Discussions/Commands/StartDiscussionHandler.php), but i don't have sure if/how that handler is called.
